EasyPost/Models/API/EndShipper.cs

```diff
@@ -90,5 +91,34 @@ public class EndShipperCollection : Collection
         internal EndShipperCollection()
         {
         }
+
+        /// <summary>
+        ///     Override the default BuildNextPageParameters method to build the parameters for the next page of EndShippers.
+        /// </summary>
+        /// <param name="entries">The entries of the <see cref="EndShipperCollection"/>.</param>
+        /// <typeparam name="T">The type of <see cref="EasyPost._base.EasyPostObject"/> the entries are. Should be <see cref="EndShipper"/>.</typeparam>
+        /// <returns>A <see cref="Dictionary{TKey,TValue}"/> of parameters to use for the subsequent API call.</returns>
+        /// <exception cref="EndOfPaginationError">Thrown if there are no more items to retrieve for the paginated collection.</exception>
+        protected internal override Dictionary<string, object> BuildNextPageParameters<T>(List<T>? entries)
+        {
+            if (entries is not List<EndShipper> endShippers)
+            {
+                throw new EndOfPaginationError();
+            }
+
+            if (endShippers == null || endShippers.Count == 0)
+            {
+                throw new EndOfPaginationError();
+            }
+
+            if (HasMore == null || !(bool)HasMore)
+            {
+                throw new EndOfPaginationError();
+            }
+
+            // TODO: How would we get the next page of EndShippers?
```

Comment:
Answer: we wouldn't. Without the ability to set `before_id` on EndShipper, people cannot currently get the next page. Since records are returned most recent to oldest, there is currently no parameter usable to get what we'd need so for EndShipper, it would just throw an error till this is patched at the API level.

Reply:
Because this isn't implemented at the API level, let's not implement anything here other than throwing an error maybe, something along the lines of "can't get next page of endshippers"